dired-sidebar.el

```diff
@@ -809,8 +809,17 @@ the relevant file-directory clicked on by the mouse."
     ;; Use `project' if `projectile' is not loaded yet.
     ;; `projectile' is a big package and takes a while to load so it's better
     ;; to defer loading it as long as possible (until the user chooses).
-    (dired-sidebar-if-let* ((pr (project-current)))
-        (project-root pr)
+    (dired-sidebar-if-let* ((pr (project-current))
+        ;; It can happen, at least in Emacs 27.1, that
+        ;; `project-current` give a non-nil result, while
+        ;; `project-root` is undefined. Fallback to assuming that the
+        ;; directory part of `project-current` is the root. See
+        ;; https://github.com/jojojames/dired-sidebar/issues/73 for
+        ;; more details.
+                            (root (if (fboundp 'project-root)
```

Comment:
Why do we need root?

Reply:
We don't. I just found it easier to read that way. I'll remove it as you wouldn't have asked the question if you did. :)